api/swagger.yaml

```diff
@@ -10491,13 +10491,9 @@ paths:
 
         ### Image tarball format
 
-        An image tarball contains one directory per image layer (named using its long ID), each containing these files:
+        An image tarball contains [Content as defined in the OCI Image Layout Specification](https://github.com/opencontainers/image-spec/blob/main/image-layout.md#content).
```

Comment:
It's probably good to use a perma-link for this, in case the URL or anchor changes; we could use the latest tag for now (`v1.1.1`); `https://github.com/opencontainers/image-spec/blob/v1.1.1/image-layout.md#content`